lib/lita/adapters/slack/api.rb

```diff
@@ -81,17 +93,21 @@ def set_topic(channel, topic)
           call_api("channels.setTopic", channel: channel, topic: topic)
         end
 
-        def rtm_start
-          response_data = call_api("rtm.start")
+        def rtm_connect
+          response_data = call_api("rtm.connect")
+          channels = conversations_list('public_channel,private_channel,im')
```

Comment:
`conversations_list` holds all the responses around public, private, and group messages (including IMs & group messages).